src/io/inputs.jl

```diff
@@ -256,3 +227,45 @@ function _yaxarray2nameddimsarray(yarray::YAXArray)::NamedDimsArray
 
     return NamedDimsArray(data; zip(dim_names, dim_labels)...)
 end
+
+"""
+    DataCube(data::AbstractArray; kwargs...)::YAXArray
+
+Constructor for YAXArray.
+"""
+function DataCube(data::AbstractArray; kwargs...)::YAXArray
+    return YAXArray(Tuple(Dim{name}(val) for (name, val) in kwargs), data)
+end
+
+"""
+    ZeroDataCube(T=Float64; kwargs...)::YAXArray
+
+Constructor for YAXArray with all entries equal zero.
+"""
+function ZeroDataCube(T=Float64; kwargs...)::YAXArray
+    return DataCube(zeros(T, [length(val) for (name, val) in kwargs]...); kwargs...)
+end
+
+function axes_names(cube::YAXArray)
+    return name.(cube.axes)
+end
```

Comment:
Looks like you're mixing in commits here - you should rebase instead of merging changes.